packages/components/src/components/PipelineRuns/PipelineRuns.test.js

```diff
@@ -72,7 +72,19 @@ it('PipelineRuns renders data', () => {
         {
           actionText: intl.formatMessage({
             id: 'test.actionText',
-            defaultMessage: 'TestAction'
+            defaultMessage: 'Stop'
```

Comment:
similar to @carlos-logro's comment
there's no need for the test to exactly mirror the actions provided in the UI
the important things would be to test:
- support for rendering multiple actions
- that when an action is clicked, the correct click handler is called

Reply:
let's revert this change, it's not adding anything to this test and the functionality is tested in the container below